package.json

```diff
@@ -32,21 +39,29 @@
     "request": "^2.69.0",
     "search-index": "0.15.0",
     "trycatch": "^1.5.21",
-    "leveldown": "^1.5.0",
-    "levelup": "^1.3.3",
-    "merge-dirs": "^0.2.1",
-    "node-minify": "^2.4.1",
-    "i18n-2": "^0.7.2"
+    "uuid": "^3.3.2"
   },
   "devDependencies": {
+    "coveralls": "^3.0.0",
+    "eslint": "^5.8.0",
+    "eslint-config-standard": "^12.0.0",
+    "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-jasmine": "^2.10.1",
+    "eslint-plugin-mocha": "^5.2.1",
+    "eslint-plugin-node": "^8.0.0",
+    "eslint-plugin-promise": "^4.0.1",
+    "eslint-plugin-standard": "^4.0.0",
     "freeport": "1.0.5",
-    "istanbul": "0.4.x",
+    "husky": "~1.1.0",
+    "istanbul": "^0.4.5",
     "jasmine": "2.9.0",
-    "nock": "3.1.0",
-    "proxyquire": "1.7.3",
-    "coveralls": "^3.0.0"
+    "lint-staged": "~7.3.0",
+    "nock": "^10.0.6",
+    "proxyquire": "1.7.3"
   },
   "scripts": {
+    "lint": "eslint ./lib --ext .js",
```

Comment:
Ideally this should be `eslint ./lib ./test --ext .js`, but the tests have a lot of errors.
Once again this is something to be fixed at some point.